packages/decentraland-ecs/src/ecs/Engine.ts

```diff
@@ -323,10 +337,12 @@ export class Engine {
     }
 
     this.simpleSystems.push(system)
+
+    return this
   }
 
   private checkRequirementsAndAdd(entity: Entity) {
-    if (!entity.isAddedToEngine()) return
+    if (!entity.isAddedToEngine()) return this
```

Comment:
```suggestion
    if (!entity.isAddedToEngine()) return 
```
Private and unnecesary